### DISCUSSION TITLE
Can't request review from members with space access?

### DISCUSSION THREAD

**Discussion by liliakai** [GitbookIO/community#738] [0. General bugs and questions]
[Documentation](https://docs.gitbook.com/content-editor/editor/change-requests#request-a-review-on-a-change-request) says that review requests will notify everyone with reviewer access or higher, but in my experience this is not happening.

I am an administrator on all my org's spaces and I have email notifications enabled and my email is verified, but I never get email notifications about review requests, and my account cannot be resolved by the requester when searching for reviewers in the review request modal.

This seems like a bug or perhaps a bug in documentation. What are the rules for populating the list of potential reviewers? How can I add someone to the space so that they will be included in review request emails?

**Comment by liliakai** [comment]
Also worth noting: when I request a review via my admin account, I can select any organization member! So maybe there is some (undocumented) filtering of the reviewer search based on the requester's permission level? Any help is appreciated.

**Reply by aikoii** [reply]
We also have an open ticket for the filtering issue, as it currently lists all users. We'll work on it 🤞

**Comment by aikoii** [comment]
Hey @liliakai! Could you share your OrgID and account (unless you're the sole admin)?

To clarify, when we say: If you don’t tag anyone in your review request, everyone with reviewer permissions or higher will get a notification, it means:

- If there are reviewers, they will all be notified
- If not, creators will be notified
- If neither, admins will be notified

Regarding the issue where your email isn’t appearing in the review request modal, we’ll need to 👀

**Reply by liliakai** [reply]
orgID WQpOq5ZFue4N6m65QCJq
my admin account which can tag reviewers DkFv6Pei9EbElfH54Z30i9N0NpI2
my test user which can't tag reviewers sJIljbKbFva9PHVVmkcbA9IcbRj1

Might the discrepancy be related to the fact that it's a "Guest" in the org?

**Reply by aikoii** [reply]
Thanks for sharing! I tested this with both accounts (I hope you don't mind 🙏).

And yes, it might be because they're Guest in the org. I'll double-check a few things and get back to you.
